packages/api/lib/writeRecords/write-granules.js

```diff
@@ -462,6 +462,14 @@ const _writeGranuleRecords = async (params) => {
         trx,
         granulePgModel,
       });
+
+      // Future: refactor to cover the entire object?
```

Comment:
This is being set because it's *possible* for _writeGranuleRecords to  to get a granule without createdAt set, have the postgres DB set the default and have it *not* set for the following writes.